Hackney.Shared.HousingSearch/Domain/Transactions/Person.cs

```diff
@@ -4,8 +4,17 @@ namespace Hackney.Shared.HousingSearch.Domain.Transactions
 {
     public class Person
     {
-        public Guid Id { get; set; }
+        public static Person Create(Guid id, string fullName)
+            => new Person(id, fullName);
 
-        public string FullName { get; set; }
+        public Person(Guid id, string fullName)
```

Comment:
Ideally, this should be made private so it can't be accessed with `new Person(...)`

Reply:
I forgot to make it private :) Will fix now